packages/@mollitia/redis/src/redis.ts

```diff
@@ -0,0 +1,77 @@
+import * as Mollitia from 'mollitia';
+import { RedisClientType, RedisModules, createClient } from 'redis';
+
+export interface CircuitStorage {
+  getState(moduleName: string): Promise<Mollitia.SerializableRecord>;
+  setState(moduleName: string, state: Mollitia.SerializableRecord[]): Promise<void>;
+  clearState(moduleName: string): Promise<void>;
+}
+
+export class RedisStorage implements CircuitStorage {
+  private client: RedisClientType<RedisModules>;
+  private prefix = 'mollitia';
+  private initializePromise: Promise<void>;
+  constructor(host: string, port: number, password = '') {
+    this.client = createClient({
+      socket: {
+        host,
+        port
+      },
+      disableOfflineQueue: true, //disableOfflineQueue should be set to true to avoid blocking requests when Redis is down
+      password
+    });
+    this.initializePromise = new Promise<void>((resolve) => {
+      this.client.on('ready', () => {
```

Comment:
Nothing to change here but I'm curious, how does the `redis` library handle errors or disconnections?
If you disable redis for a while, does it try to reconnect automatically?
If there's configuration options, should we expose to consumer?